package.json

```diff
@@ -1,9 +1,17 @@
 {
   "name": "@aligent/ts-code-standards",
-  "main": "src/index.js",
   "type": "module",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.mjs"
+    },
+    "./tsconfigs-base": "./tsconfigs/base.json",
+    "./tsconfigs-react": "./tsconfigs/react.json"
+  },
   "scripts": {
     "test": "tsc && eslint .",
+    "build": "npx rollup -c",
```

Comment:
@kai-nguyen-aligent Could we change this one to?:

```bash
pnpm exec rollup -c
```

I'd prefer to avoid `npx` commands where possible because:

- It can get a bit confusing which version is being used.
- It's possible to end up with a disconnect with what's explicitly installed as a dev dependency and what gets used via NPX.